BSM/Chapter_03_00_Bayesian_CLT.ipynb

```diff
@@ -0,0 +1,92 @@
+{
```

Comment:
<p>I like the combining into one. Makes things easier to read</p>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>